### ISSUE TITLE
Dark Sky's API is not available anymore

### ISSUE THREAD

**Issue by wall03** [notkanbaru/maki#4]
Dark Sky's API is not available anymore. I don't use the bot but either your README needs to be updated or both.

> make sure you have an env file in your root project directory that looks like this, obviously with everything filled out:
> 
> 
> PREFIX=
> BOT_TOKEN=
> BING_MAPS_KEY=
> **DARK_SKY_KEY=**

sorry if you already fixed this. Just wanted to bring this to your attention

**Comment by espeon** [comment]
yeah, i'm aware. i'm planning to switch to climacell once i get some small things worked out (namely attempting to recreate the weather sentence dark sky gives you)
